example/src/main/java/io/netty/example/http/websocketx/server/WebSocketFrameHandler.java

```diff
@@ -21,15 +21,15 @@
 import io.netty.channel.SimpleChannelInboundHandler;
 import io.netty.handler.codec.http.websocketx.TextWebSocketFrame;
 import io.netty.handler.codec.http.websocketx.WebSocketFrame;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import io.netty.util.internal.logging.InternalLogger;
```

Comment:
Do we really want an example to use an internal class? Should we just add the slf4j dependency to the example instead?

Reply:
@ejona86 good question... maybe we should just not log at all ?